src/linear_algebra/tensor_product.lean

```diff
@@ -376,6 +376,21 @@ begin
   exact submodule.span_mono (λ _ ⟨m, n, h⟩, ⟨m, n, trivial, trivial, h⟩),
 end
 
+lemma every_element_pure_if_absorbing_add (x : tensor_product R M N)
```

Comment:
Did we end up getting this lemma in another PR? I think I remember seeing it before.

Reply:
you probably saw this before because I mentioned it on zulip.
I created a thread about this because I wasn't sure about whether to add a lemma in my PR if it wasn't needed anymore, and the conclusion was that the lemma might still be useful (which is why I'm including it)